WebRoot/js/ajax/dwt/xforms/OSelect_XFormItem.js

```diff
@@ -1078,7 +1079,7 @@ OSelect_Check_XFormItem.prototype.getChoiceHTML = function (itemNum, value, labe
                         (this.getIsEnabled())?(" ondblclick=\"" + ref + ".onChoiceDoubleClick(" + itemNum + ", event||window.event)\""):"",
```

Comment:
Can we remove the commented out code in lines 1050-1066 (Bug 44925) as it still has the code without XSS fix and I see no good reason to continue to leave some old, broken code commented out in here.